libs/remix-ui/debugger-ui/src/lib/slider/slider.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react'
+
+export const Slider = ({ stepManager, sliderValue }) => {
+  const [state, setState] = useState({
+    currentValue: ''
+  })
+
+  useEffect(() => {
```

Comment:
Each time the slider is moved, line 9 get hit 1 time and line 13 2 times.
even if `if (value === state.currentValue) return` helps, that may be useful to check if that may cause performance issue in the stack.